package.json

```diff
@@ -136,7 +136,6 @@
     "ajv-merge-patch": "^5.0.1",
     "branch-comparer": "^1.1.0",
     "cors": "^2.8.5",
-    "coveralls": "^3.1.1",
```

Comment:
We need to remove the badge in the readme.md file as well

Reply:
Fixed.